k-distribution/pl-tutorial/2_languages/2_kool/1_untyped/kool-untyped.md

```diff
@@ -238,7 +238,7 @@ syntax includes:
 
 Old desugaring rules, from SIMPLE
 ```k
-  rule if (E) S => if (E) S else {}                                 [macro]
+  rule if (E) S => if (E) S else {}
   rule for(Start Cond; Step) {S} => {Start while (Cond) {S Step;}}  [macro]
   rule var E1::Exp, E2::Exp, Es::Exps; => var E1; var E2, Es;       [macro-rec]
```

Comment:
It seems we should remove `macro` from the `for` rule as well?

Reply:
Removed it.
For this PR I focused more on `regression-new` since the changes to the pl-tutorial would get pretty involved.
For example in kool-untyped `var` becomes a bit trickier since macros should disappear after they are applied, but here we're using macros for unwrapping multiple declarations.
That would require a bit of a rewrite, checking comments, and I'm not sure I should be doing these changes since I'm not the owner of the definition.